test/multicaller.ts

```diff
@@ -40,3 +40,26 @@ multi.execute().then((result) => {
   }
   */
 });
+
+multi.execute2(undefined, true).then((result) => {
```

Comment:
@abigger87 Sorry for slow review! I just tested this and for me it fail with this error:
```
Error: cannot estimate gas; transaction may fail or may require manual gas limit (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":43,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n"```

Are you able to reproduce this error? When execute function is called it reset the multicaller stored calls, you may need to add calls again or copy "multi" before calling execute()

Reply:
Nope, it's not obvious to me how to test/reproduce this. Can you share the command you ran?

Reply:
Will make your suggestions anyway!

Reply:
@abigger87 To test it you'll need to run the `build` script then `node ./test/multicaller.ts`. When I test again now with the changes it still give me error. How do you test?

Reply:
I'm getting an unpredictable gas limit error when I run `node ./test/multicaller.ts` but don't see any environment variable specification anywhere for an RPC. Is there any other config I need to do to get this to run? Open to taking this to pms/dms I'm on TG @andreaslb or twitter @andreasbigger 

Here's the error:

<img width="983" alt="Screen Shot 2022-03-04 at 12 51 22 PM" src="https://user-images.githubusercontent.com/21288394/156839985-3023c907-3e94-4b71-8ae7-6734d83cb5db.png">

Reply:
Seems it's an issue with`networks.json` which I didn't change except for adding the multicall2 addresses?

Reply:
Ok fixed. Bad multicall address.
When you get a chance can you give it a shot @bonustrack - Am getting a call exception, but all the parameters seem correct to me 🤔 

Here's `Multicall2` on mainnet: https://etherscan.io/address/0x5ba1e12693dc8f9c48aad8770482f4739beed696#code

<img width="982" alt="Screen Shot 2022-03-04 at 2 21 12 PM" src="https://user-images.githubusercontent.com/21288394/156849840-122ecb30-1e3d-4261-a568-063d8b539178.png">